docs/server/project_setup.rst

```diff
@@ -4,19 +4,25 @@ Create a Project
 ================
 
 Now that you have the server up and running, this guide will help you get
-started with Pootle setup for getting translations.
+started on how enable translations on the Pootle server.
```

Comment:
...will help you to setup translations on the server.

I'd check to see if @jason-p-pickering change might have included such linguistic fixes.